projects/vdk-plugins/vdk-trino/src/vdk/plugin/trino/trino_config.py

```diff
@@ -1,4 +1,93 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+from typing import cast
+
+from vdk.internal.core.config import Configuration
 
 trino_templates_data_to_target_strategy: str = ""
+
+
```

Comment:
Extract the strings as constants.
```suggestion
TRINO_HOST = "TRINO_HOST"
TRINO_PORT = "TRINO_PORT"
...etc
```

Reply:
Yes, good point. thanks.